src/main/java/gov/usds/case_issues/config/RetryableDataSourceBeanPostProcessor.java

```diff
@@ -0,0 +1,72 @@
+package gov.usds.case_issues.config;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import javax.sql.DataSource;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.jdbc.datasource.AbstractDataSource;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Component;
+
+/**
+ * Wrap any {@link DataSource} bean in retry configuration.
+ */
+@Order(Ordered.HIGHEST_PRECEDENCE)
+@Component
+@Profile("db-dockerized")
```

Comment:
I was going to make all of these strings into string constants at one point a few months ago. Still probably worth doing.

Reply:
created a ticket for this. seems out of the scope of this PR https://trello.com/c/VzJu9e8h